src/pages/PersonalInfo/PersonalInfo.tsx

```diff
@@ -1,8 +1,33 @@
+import { useState, useEffect } from 'react';
+import styles from './PersonalInfo.module.css'
+import inter from './'
+
 const PersonalInfo = () => {
   // Personal info tab
   console.log("PersonalInfo");
+  const [tabIndex, setTabIndex] = useState(0);
```

Comment:
Set the default as one
```suggestion
  const [tabIndex, setTabIndex] = useState(1);
```